README.md

```diff
@@ -84,9 +85,14 @@ variables might be required for proper scaling and resolutions:
 | BALENA_HOST_CONFIG_hdmi_mode          | 87                 |
 | BALENA_HOST_CONFIG_rotate_screen      | 1                  |
 
+### D-Bus
+
+The D-Bus container exists to enable the optional display on your Pi-hole device for [PADD](https://github.com/pi-hole/PADD).
+
+It is expected behavior for this container to run once upon boot and then exit.  It will remain in `Exited` status during normal operation.
```

Comment:
Nitpick, but missing an empty line between this and the next header.

Reply:
Nitpicks are good.  Fixing.